portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -83,9 +76,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       response.sendRedirect("/index.html");
   }
 
-  private String convertToJsonUsingGson(ArrayList<Comment> comments) {
+  private String convertToJsonUsingGson(ArrayList<Comment> comments, int numComments) {
     Gson gson = new Gson();
-    String json = gson.toJson(comments);
+
+    JsonObject obj = new JsonObject();
+    String jsonComments = gson.toJson(comments);
+    obj.addProperty("comments", jsonComments);
+    obj.addProperty("numComments", numComments);
+    String json = gson.toJson(obj);
```

Comment:
This part is good as is, we just need to make sure only the comments we want to display are being passed in.